kie-internal/src/main/java/org/kie/internal/task/api/TaskQueryService.java

```diff
@@ -29,102 +29,104 @@
 
     List<TaskSummary> getTasksAssignedAsPotentialOwnerByStatus(String userId, List<Status> status);
-    
+
     List<TaskSummary> getTasksAssignedAsPotentialOwnerByStatusByGroup(String userId, List<String> groupIds, List<Status> status);
-    
+
     List<TaskSummary> getTasksAssignedAsRecipient(String userId);
 
     List<TaskSummary> getTasksAssignedAsTaskInitiator(String userId);
 
     List<TaskSummary> getTasksAssignedAsTaskStakeholder(String userId);
 
     List<TaskSummary> getTasksAssignedByGroup(String groupId);
-    
+
     List<TaskSummary> getTasksAssignedByGroups(List<String> groupsId);
-    
+
     List<TaskSummary> getTasksAssignedByGroupsByExpirationDate(List<String> groupIds,  Date expirationDate);
-    
+
     List<TaskSummary> getTasksAssignedByGroupsByExpirationDateOptional(List<String> groupIds,  Date expirationDate);
-    
+
     List<TaskSummary> getTasksOwned(String userId);
 
     List<TaskSummary> getTasksOwnedByStatus(String userId, List<Status> status);
-    
+
     List<TaskSummary> getTasksOwnedByExpirationDate(String userId, List<Status> status, Date expirationDate);
-    
+
     List<TaskSummary> getTasksOwnedByExpirationDateOptional(String userId, List<Status> status, Date expirationDate);
-    
+
     List<TaskSummary> getTasksOwnedByExpirationDateBeforeSpecifiedDate(String userId, List<Status> status, Date date);
-    
+
     List<TaskSummary> getSubTasksAssignedAsPotentialOwner(long parentId, String userId);
-    
+
     List<TaskSummary> getSubTasksByParent(long parentId);
-    
+
     List<TaskSummary> getTasksByStatusByProcessInstanceId(long processInstanceId, List<Status> status);
 
     List<TaskSummary> getTasksByStatusByProcessInstanceIdByTaskName(long processInstanceId, List<Status> status, String taskName);
 
     int getPendingSubTasksByParent(long parentId);
-    
+
     Task getTaskByWorkItemId(long workItemId);
-    
+
     Task getTaskInstanceById(long taskId);
-    
+
     List<Long> getTasksByProcessInstanceId(long processInstanceId);
-    
+
     Map<Long, List<OrganizationalEntity>> getPotentialOwnersForTaskIds(List<Long> taskIds);
-  
+
     int getCompletedTaskByUserId(String userId);
     int getPendingTaskByUserId(String userId);
 
     /**
      * This method will no longer be available as of jBPM 7.x
-     * 
+     *
      * @param parameters A map of parameters to add to the query
      * @param union Whether the query should return a union or intersection of the criteria (parameters)
-     * 
+     *
      * @see {@link #query(String, QueryData)}
      * @return
      */
     @Deprecated
     List<TaskSummary> getTasksByVariousFields( String userId, Map<String, List<?>> parameters, boolean union);
-    
-    
+
+
     /**
-     * 
+     *
      * @param queryWhere
      * @return
      */
     List<TaskSummary> query( String userId, Object queryWhere );
+
+    List<TaskSummary> getTasksByPotentialOwnerGroups( List<String> groupIds );
```

Comment:
This method uses plural (groups)and the other method above (`getTasksByGroup`) uses singular (group). Is that on purpose? (e.g. due to being consistent on within the different interfaces, even though those are consistent between itself)